mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/KafkaSyncService.kt

```diff
@@ -39,4 +43,29 @@ class KafkaSyncService(
 
         logger.info("Oppdaterte status for $count tiltaksgjennomføringer")
     }
+
+    fun oppdaterTiltakstypestatus(today: LocalDate, lastSuccessDate: LocalDate) {
+        var offset = 1
+        var count = 0
+
+        logger.info("Oppdaterer statuser for tiltakstyper med start eller sluttdato mellom $lastSuccessDate og $today")
+        do {
```

Comment:
Denne  `do { ... } while`-runtinen er kanskje en kandidat for en slags `paginate()` database utility?